.github/workflows/push-pull.yml

```diff
@@ -0,0 +1,81 @@
+name: Pull/push
+on:
+  # Trigger the workflow on push or pull request,
+  # but only when targeting the main branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build_ibamr:
+    runs-on: ubuntu-latest
```

Comment:
should this be fedora?

Reply:
No, this is the host OS that the container runs in. Github does not provide fedora, only ubuntu.